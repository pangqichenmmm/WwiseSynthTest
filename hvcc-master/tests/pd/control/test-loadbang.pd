#N canvas 261 422 361 90 10;
#X declare -path abs;
#X obj 10 17 loadbang;
#X obj 71 17 declare -path abs;
#X obj 10 39 print main;
#X obj 77 39 abs_loadbang;
#X text 9 60 ensure that loadbangs in deeper abstractions fire first
;
#X connect 0 0 2 0;
