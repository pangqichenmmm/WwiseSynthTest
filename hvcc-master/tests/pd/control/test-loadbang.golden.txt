[@ 0.000] abs: bang
[@ 0.000] main: bang
